### PR TITLE
Update Juggernaut

### PR DESCRIPTION
**Pull request by vilaboim**
I updated Juggernaut within the juggernaut-demo and then had to change several files that were not following our standards.
There are many other files to update, but we can do that later because this PR is already huge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,31 +2,60 @@
...
     "prop-types": "^15.7.2",
-    "qs": "^6.6.0",
-    "react": "^16.8.2",
-    "react-dom": "^16.8.2",
-    "react-router-dom": "^5.0.0",
-    "react-scripts": "3.0.0",
-    "styled-components": "^4.1.3",
-    "yup": "^0.27.0"
+    "qs": "^6.9.1",
+    "react": "^16.12.0",
```
Comment (fredericofigueira) [inline package.json:15]: I think we can go to React 17, what do you think about it?

File: package.json [hunk 2]
```diff
@@ -2,31 +2,60 @@
...
+    "react-dom": "^16.12.0",
+    "react-i18next": "^11.7.3",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "^3.4.3",
```
Comment (fredericofigueira) [inline package.json:19]: May we update React-Scripts for the latest 4.0.1?

File: src/containers/SignInSide/index.jsx [hunk 3]
```diff
@@ -66,14 +61,14 @@ function SignInSide() {
               </Grid>
               <Grid item>
                 <Link href="#" variant="body2">
-                  Don't have an account? Sign Up
+                  Don`t have an account? Sign Up
```
Comment (fredericofigueira) [inline src/containers/SignInSide/index.jsx:64]: I think the right way to write this sentence is `Don't have an account? Sign Up`

**Review by fredericofigueira — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vilaboim) [reply package.json:15]: Material-Ui v4 doesn't have official support for React 17 like v5 has.

File: package.json (hunk 2 above)
Comment (vilaboim) [reply package.json:19]: You're right, but I think is better to do this in another PR.

File: src/containers/SignInSide/index.jsx (hunk 3 above)
Comment (vilaboim) [reply src/containers/SignInSide/index.jsx:64]: I've changed this because ESLint was accusing an error, but I'll try another solution.

**Review by fredericofigueira — APPROVED** [review]

File: src/components/Form/Container.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
+
+const Container = styled.div`
```
[the remark this answers is not in the document]
Comment (vilaboim) [reply src/components/Form/Container.jsx:3]: Esse código se repetia para todos os forms, então criei o componente.
Ele tem esse nome para não ficar repetindo a palavra `Form` no import, mas para não causar confusão usei ele assim:
`import FormContainer from 'components/Form/Container'`

**Review by douglaszaltron — APPROVED** [review]

**Review by douglaszaltron — APPROVED** [review]

**Pull request merged**